pkg/kancontroller/kancontroller.go

```diff
@@ -83,6 +71,11 @@ func Execute() {
 		return
 	}
 
+	err = handler.RunWebhookServer()
+	if err != nil {
+		log.WithError(err).Print("Failed to start kanister controller server")
```

Comment:
```suggestion
		log.WithError(err).Print("Failed to start validating webhook server")
```
Any reason for not specifying this?
And should we return here?

Reply:
apart from webhook there were some other endpoints that we register that is the reason I said kanister controller. But I think validating webhook controller is better. I can do that.